neurokit2/rsp/rsp_eventrelated.py

```diff
@@ -130,6 +133,7 @@ def _rsp_eventrelated_amplitude(epoch, output={}):
         signal = epoch["RSP_Amplitude"][epoch.index > np.min(epoch.index)].values
 
     # Max / Min / Mean
+    output["RSP_Amplitude_Baseline"] = epoch["RSP_Amplitude"].iloc[zero]
```

Comment:
since you compute the baseline above why don't you re-use this variable?

Reply:
They're different, 'baseline' variable is computed for all time points before stimulus onset (t=0) whereas rsp_amplitude_baseline is essentially the value closest to t=0

Reply:
riiight. which maeks me think, what's the default epoch window used for ecg_ and rsp_ analysis (and the one that we use in cocon?)

Reply:
I wonder whether we should use the "closest to 0" adjustment method for the mean/max/min parameters; because it's based on interpolated signal so the real baseline is the value at 0, averaging some time before makes little sense (the pre-0 window can be used for stuff like detrending or whatnot, but for baseline correction I'd say we just correct with the closest to 0 value)

Reply:
> riiight. which maeks me think, what's the default epoch window used for ecg_ and rsp_ analysis (and the one that we use in cocon?)

There were two window lengths we used from what we discussed last time, so shorter window is -0.5 to 1.5 for both ECG and RSP, and longer window is -0.5 to 2.25s for ECG and -0.5 to 3s for RSP.

> I wonder whether we should use the "closest to 0" adjustment method for the mean/max/min parameters; because it's based on interpolated signal so the real baseline is the value at 0, averaging some time before makes little sense (the pre-0 window can be used for stuff like detrending or whatnot, but for baseline correction I'd say we just correct with the closest to 0 value)

Yes I agree, for our rate features the mean/max/min parameters are corrected based on this exact 'closest to 0' value, wonder why we didn't do this for our amplitude features 🤔

Reply:
@DominiqueMakowski bump ^^

Reply:
yeah we should be consistent and hte the closest to 0 point